docs/changelog.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 * :feature:`-` Users can now manually reset cached Beefy Finance vault data to trigger re-pulling the data during decoding.
+* :bug:`-` Fix an issue where Paraswap swaps were not being decoded properly when curve pool interactions were present in the same transaction.
```

Comment:
User facing changelogs please.

`Paraswap swaps will be properly decoded when curve pool interactions are present in the same transaction`

Same for the rest of the "Fix" ones.